lib/oli_web/templates/page_delivery/exploration.html.heex

```diff
@@ -0,0 +1,20 @@
+
+<Components.Delivery.NavSidebar.main_with_nav {assigns}>
+  <div class="relative flex-1 flex flex-col pb-[60px]">
+    <%= render OliWeb.LayoutView, "_pay_early.html", assigns %>
+
+    <Components.Delivery.ExplorationShade.exploration_shade />
+
+    <div class="container mx-auto px-10 mt-3 mb-5 flex flex-col">
+      <%= live_render @conn, Components.Delivery.ExplorationList, session: %{ "section_slug" => @section_slug } %>
+    </div>
+
+    <%= render OliWeb.LayoutView, "_delivery_footer.html", assigns %>
+  </div>
+</Components.Delivery.NavSidebar.main_with_nav>
+
+<%= if Oli.Utils.LoadTesting.enabled?() do %>
+  <!--
+__OVERVIEW_PAGES__<%= encode_pages(@conn, @section_slug, @hierarchy) %>__OVERVIEW_PAGES__
+-->
+<% end %>
```

Comment:
What's this for?

Reply:
Forget to delete it. There is no sense in that code commented there 😅